# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1536838416 14400
#      Thu Sep 13 07:33:36 2018 -0400
# Node ID c25ee75beea1cfaf39275779fbc825fa8c17c9b8
# Parent  bb4af245dff79d6d52bafdf26578506158cae198
adjust tests for menu changes in changeset eba6d823cf62

* allchild.m, findall.m: Fix tests for new menu layout used in fltk
figure windows.

diff --git a/scripts/plot/util/allchild.m b/scripts/plot/util/allchild.m
--- a/scripts/plot/util/allchild.m
+++ b/scripts/plot/util/allchild.m
@@ -49,16 +49,16 @@ endfunction
 
 
 %!testif HAVE_OPENGL, HAVE_FLTK; have_window_system ()
 %! toolkit = graphics_toolkit ("fltk");
 %! hf = figure ("visible", "off");
 %! unwind_protect
 %!   l = line;
 %!   kids = allchild (hf);
-%!   assert (get (kids, "type"), {"axes"; "uimenu"; "uimenu"});
+%!   assert (get (kids, "type"), {"axes"; "uimenu"; "uimenu"; "uimenu"});
 %! unwind_protect_cleanup
 %!   close (hf);
 %!   graphics_toolkit (toolkit);
 %! end_unwind_protect
 
 %!error allchild ()
 %!error allchild (1, 2)
diff --git a/scripts/plot/util/findall.m b/scripts/plot/util/findall.m
--- a/scripts/plot/util/findall.m
+++ b/scripts/plot/util/findall.m
@@ -51,14 +51,14 @@ endfunction
 
 
 %!testif HAVE_OPENGL, HAVE_FLTK; have_window_system ()
 %! toolkit = graphics_toolkit ("fltk");
 %! hf = figure ("visible", "off");
 %! unwind_protect
 %!   h = findall (hf);
 %!   all_handles(1) = {"figure"};
-%!   all_handles(2:18,1) = {"uimenu"};
+%!   all_handles(2:24,1) = {"uimenu"};
 %!   assert (get (h, "type"), all_handles);
 %! unwind_protect_cleanup
 %!   close (hf);
 %!   graphics_toolkit (toolkit);
 %! end_unwind_protect
